app/src/components/HomePageJN/Hero.tsx

```diff
@@ -48,6 +55,10 @@ export function Hero({
     skip: !city?.locode,
   });
 
+  const formattedEmissions = ghgiCityData?.totalEmissions
+    ? formatEmissions(ghgiCityData.totalEmissions)
+    : { value: t("N/A"), unit: "" };
```

Comment:
I think we are using the translation key `n-a` elsewhere, might make sense to use it here as well.